src/main/fc/config.c

```diff
@@ -78,7 +78,7 @@ PG_RESET_TEMPLATE(systemConfig_t, systemConfig,
     .activeRateProfile = 0,
     .debug_mode = DEBUG_MODE,
     .task_statistics = true,
-    .cpu_overclock = false,
+    .cpu_overclock = 0,
```

Comment:
An enum for the various speed settings would make the code a lot more self explanatory.

Reply:
Agreed, though the numerical value is not used directly unless it's 0.

Reply:
Good point.